index.bs

```diff
@@ -566,12 +568,13 @@ The <dfn method for=CookieStore>get(|name|)</dfn> method steps are:
 <div algorithm>
 The <dfn method for=CookieStore>get(|options|)</dfn> method steps are:
 
-1. Let |origin| be the [=current settings object=]'s [=/origin=].
+1. Let |settings| be [=/this=]'s [=/relevant settings object=].
+1. Let |origin| be |settings|'s [=environment settings object/origin=].
 1. If |origin| is an [=opaque origin=], then return [=a promise rejected with=] a "{{SecurityError}}" {{DOMException}}.
-1. Let |url| be the [=current settings object=]'s [=creation URL=].
+1. Let |url| be |settings|'s [=environment/creation URL=].
 1. If |options| is empty, then return [=a promise rejected with=] a {{TypeError}}.
 1. If |options|["{{CookieStoreGetOptions/url}}"] is present, then run these steps:
-    1. Let |parsed| be the result of [=basic URL parser|parsing=] |options|["{{CookieStoreGetOptions/url}}"] with [=/this=]'s [=relevant settings object=]'s [=API base URL=].
+    1. Let |parsed| be the result of [=basic URL parser|parsing=] |options|["{{CookieStoreGetOptions/url}}"] with |settings|'s [=environment settings object/API base URL=].
     1. If the [=current global object=] is a {{Window}} object and |parsed| does not [=url/equal=] |url|,
```

Comment:
Does this need to be `[=relevant global object=]`?

Reply:
The bikeshed source makes this confusing here. The _settings_ variable _is_ the relevant settings object (for _this_) captured in step 1 of the algorithm, which is an environment settings object. The `[=environment settings object/API base URL=]` reference is to the definition of API base URL for an environment settings object.

In pseudo C++ this would be:

EnvironmentSettingsObject* settings = this->RelevantSettingsObject();
...
auto parsed = URL::Parse(options["url"], settings->APIBaseURL());

Reply:
My question was mainly around the `If the [=current global object=] is a {{Window}}` section, since `current global object` definition mentioned the `current realm`. But reading this better, it's just used for comparison so seems like my comment is not relevant. Sorry if I miss understood something here.

Reply:
Oh, sorry, I was looking at the changes line only. I'll review again.